composer.json

```diff
@@ -39,7 +39,7 @@
         "doctrine/persistence": "^2",
         "twig/twig": "^2.13|^3.0.4",
         "psr/cache": "^1.0|^2.0",
-        "psr/container": "^1.0",
+        "psr/container": "^1.1|^2.0",
```

Comment:
Because DI does not support 2.0, this should actually be "^1.1.1" as in the DependencyInjection `composer.json` file.